tests/test_models.py

```
@@ -2,772 +2,228 @@
+def test_validation_mixin_valid_inputs(
+    field: str, value: Union[str, List[str]], expected_value: Any, description: str
+) -> None:
+    """Test ValidationMixin with valid inputs."""
+    data: TestModelDict = {"mac": VALID_MAC}  # Base required field
+    data[field] = value  # type: ignore
+    model = TestModel(**data)
+    assert getattr(model, field) == expected_value, description
+
+
+@pytest.mark.parametrize(
+    "field,invalid_value,error_pattern",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)